HousingSearchApi/data/elasticsearch/assetIndex.json

```diff
@@ -1,109 +1,74 @@
 {
-  "aliases": {
-
-  },
   "mappings": {
     "properties": {
       "id": {
-        "type": "text",
-        "fields": {
-          "keyword": {
-            "type": "keyword",
-            "ignore_above": 256
-          }
-        }
+        "type": "keyword"
```

Comment:
Key change 1: ID is keyword (good for exact matching) instead of text (good for partial / fuzzy matching)